oxrdflib/parser.py

```diff
@@ -0,0 +1,121 @@
+import warnings
+from typing import Any, Optional, Union
+
+from rdflib import ConjunctiveGraph, Graph
+from rdflib.exceptions import ParserError
+from rdflib.parser import FileInputSource, InputSource, Parser
+
+from oxrdflib._converter import to_ox
+from oxrdflib.store import OxigraphStore
+
+__all__ = [
+    "OxigraphTurtleParser",
+    "OxigraphNTriplesParser",
+    "OxigraphRdfXmlParser",
+]
+
+
+class OxigraphParser(Parser):
+    def __init__(self):
+        pass
+
+    def parse(
+        self,
+        source: InputSource,
+        sink: Union[Graph, ConjunctiveGraph],
```

Comment:
```suggestion
        sink: Graph,
```

(ConjunctiveGraph is a subclass of Graph)